lib/trackler/track.rb

```diff
@@ -81,8 +81,11 @@ def test_pattern
       end
     end
 
-    def docs(image_path = DEFAULT_IMAGE_PATH)
-      OpenStruct.new(docs_by_topic(image_path))
+    def docs(positional_image_path_which_is_deprecated = nil, image_path: nil)
```

Comment:
`positional_image_path_which_is_deprecated` is perhaps the best name, ever. I love how forward-thinkingly helpful it is 🌻